src/core/pd_logger.js

```diff
@@ -0,0 +1,27 @@
+import winston from "winston";
+class Logger {
+  constructor() {
+    this.logger.add(
+      new winston.transports.Console({
+        format: winston.format.combine(
+          winston.format.timestamp(),
+          winston.format.colorize(),
+          winston.format.printf(
+            ({ level, message, timestamp }) =>
+              `${timestamp} [${level}]: ${message}`
```

Comment:
share sample logger here how it will look

Reply:
as a comment right??